pkgs/tools/misc/datefmt/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "datefmt";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "http://cdn.jb55.com/tarballs/datefmt/datefmt-${version}.tar.gz";
+    sha256 = "5d5e765380afe39eb39d48f752aed748b57dfd843a4947b2a6d18ab9b5e68092";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
```

Comment:
You could add `testVersion` (implemented in pkgs/build-support/trivial-builders.nix) with `datefmt, testVersion` in scope:
```
passthru.tests = testVersion { package = datefmt; };
```